ocaml/xenopsd/dbgring/dbgring.ml

```diff
@@ -11,27 +11,29 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-open Xenops_utils
 
 let xenstored_proc_port = "/proc/xen/xsd_port"
 
 let xenstored_proc_kva = "/proc/xen/xsd_kva"
 
+let gnttab = Gnt.Gnttab.interface_open ()
+
```

Comment:
Should this use `Fun.protect`?

Reply:
There is a 'with' function I could use inside Gnttab, but I'm not sure how safe it is to close the gnt when the mapping is still in use, I'll have to test it.